hudi-common/src/main/java/org/apache/hudi/common/model/OverwriteWithLatestAvroPayload.java

```diff
@@ -36,6 +36,9 @@
 public class OverwriteWithLatestAvroPayload extends BaseAvroPayload
     implements HoodieRecordPayload<OverwriteWithLatestAvroPayload> {
 
+  public static final String DEFAULT_DELETE_FIELD = "_hoodie_is_deleted";
```

Comment:
sorry, don't understand why we need this. the config value will either refer to "_hoodie_is_deleted" if not set, or will refer to the user defined prop. So, we could initialize isDeletedField = null and set it in constructor. correct me if my understanding is wrong.

Reply:
Remove DEFAULT_DELETE_FIELD.